src/RicardoFontanelli/LaravelTelegram/Telegram.php

```diff
@@ -222,10 +222,13 @@ public static function __callStatic($name, $arguments)
     {
         (!isset($this->resources[$name])) ? $this->resources[] = $name : null;
 
+        if (count($arguments) == 1 && is_array($arguments[0])) {
+            $arguments = $arguments[0];
+        }
+
         $method = isset($arguments['method']) ? $arguments['method'] : 'POST';
         unset($arguments['method']);
-        $params = json_encode($arguments);
-        return $this->callAPI('POST', $name, json_encode($arguments));
```

Comment:
`$params` seems to be useless.

`__callAPI` expects an array in the 3rd parameter "to be converted to JSON" not "already converted"